src/setCompliment.js

```diff
@@ -0,0 +1,9 @@
+export default function setCompliment(arr1, arr2){
+  const concatSortArray = arr1.concat(arr2).sort()
+  const results = []
+  for (var i=0; i<concatSortArray.length; i++){
+    arr1.includes(concatSortArray[i])
```

Comment:
you're not using the return value of this expression so an if would be better than an ternary. 
```js
const member = concatSortArray[i]
if (!arr1.includes(member) results.push(member)
```